### ISSUE TITLE
Run without debugging (Ctrl+F5)

### ISSUE THREAD

**Issue by chapalee** [Autodesk-AutoCAD/AutoLispExt#98]
**Describe the bug**
Run without debugging (Ctrl+F5) It doesn't work for me in AutoLISP.

**To Reproduce**
When trying to Run without debugging, the application still runs.

**Desktop (please complete the following information):**
 - OS: Window
 - Version 10

**Additional context**
This is a mistake?
Or am I doing something wrong?

**Comment by rtogores** [comment]
Yes, I have noticed that too.

**Comment by Sen-real** [comment]
Hi Chapalee,
do you mean that with Ctrl+F5, it's still actually debugging the LSP file?

**Comment by chapalee** [comment]
Exactly, the debugging begins and it shouldn't be like that.
He should connect to autocad but not debug.

**Comment by Sen-real** [comment]
Thanks a lot for logging this issue. I totally agree with you.

**Comment by rtogores** [comment]
More on this. If I have a breakpoint in my code "Run without debugging" will stop anyway at the breakpoint, just as if I was using plain F5.
I suppose the idea was that it should run without stopping at the breakpoint. Or no?

**Comment by Sen-real** [comment]
> More on this. If I have a breakpoint in my code "Run without debugging" will stop anyway at the breakpoint, just as if I was using plain F5.
> I suppose the idea was that it should run without stopping at the breakpoint. Or no?

Yes I agree with you. It should just connect to ACAD and load the LSP file.

**Comment by 57gmc** [comment]
Still a bug in 2024.

**Comment by Sen-real** [comment]
> Still a bug in 2024.

Thanks for your reminder.

BTW, I think this menu item is for the launch mode (i.e. to start a new Autocad instance to run/debug the active lisp file). So in addition to support 'Launch Without Debugging' for the launch mode, it would be better to disable the 'Launch Without Debugging' entry in the attach use case.

**Comment by 57gmc** [comment]
> it would be better to also make it impossible to choose the 'attach' mode in the 'Run Without Debugging' use case.
Are you suggesting something I need to do or just a placeholder for a :ToDo?

**Comment by Sen-real** [comment]
> Are you suggesting something I need to do or just a placeholder for a :ToDo?

Yeah just adding a :ToDo item for this issue. Let me update my comment.

**Comment by carlosmgilp** [comment]
Still a bug in AutoCAD 2025
